exonum-light-client/src/main/lombok/com/exonum/client/response/HealthCheckInfo.java

```diff
@@ -22,7 +22,8 @@
 @Value
 public class HealthCheckInfo {
   /**
-   * Shows information about whether it is possible to achieve the consensus between
+   * Consensus status which shows an information about
```

Comment:
Could we shorten to "Consensus status shows/indicates whether it it is possible to achieve the consensus between validators in the current network state."?